open_event/api/speakers.py

```diff
@@ -55,6 +56,13 @@ def get(self, event_id, speaker_id):
         """Fetch a speaker given its id"""
         return DAO.get(event_id, speaker_id)
 
+    @requires_auth
+    @api.doc('delete_speaker')
+    @api.marshal_with(SPEAKER)
+    def delete(self, event_id, speaker_id):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2716743259/issues/source?bid=3233864&fileBranchId=3345807#l62)